docs/DeveloperGuide.md

```diff
@@ -320,7 +364,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case resumes at step 2.
 
-**Use case: UC02 - Delete a student**
+**Use case: UC04 - Find a student**
+
+**MSS**
+
+1.  User requests to find a student.
+2.  User provides the parameters to find student.
+3.  Tutor's Pet finds the student.
+4.  Student is found by Tutor's Pet.
```

Comment:
Technically Tutor's Pet can return more than one student found, so maybe it would be good to mention that it shows all matching students.

Reply:
Applies to finding classes as well.

Reply:
Resolved by rewording expression to show that Tutor's Pet can return more than one user found